find/scripts/extract_download_logs.py

```diff
@@ -74,23 +72,36 @@ def extract_ids_from_message(message: List[List[dict]]) -> List[str]:
 
 def fetch_user_data(user_ids: List[str]) -> dict:
```

Comment:
Can we call this `get_email_addresses_for_user_ids` or something more descriptive?

Reply:
Changed the function name.